### PR TITLE
RPRBLND-1861: Add "Help/About" UI menu to USD.

### PR DESCRIPTION
**Pull request by DagerD**
### PURPOSE
Add "Help/About" UI menu to USD

### EFFECT OF CHANGE
Added "Help/About" UI menu

### TECHNICAL STEPS
Added corresponding panel and operator

### CODE REVIEW

**Review by VascoPi — APPROVED** [review]
Works fine

File: src/hdusd/ui/render.py [hunk 1]
```diff
@@ -13,11 +13,45 @@
...
+from . import HdUSD_Panel, HdUSD_Operator
+
+from hdusd import bl_info
 
 from pxr import UsdAppUtils, Tf
 
+from rpr import RprUsd
+
+class HDUSD_OP_open_web_page(HdUSD_Operator):
+    '''
```
Comment (bnagirniak) [inline src/hdusd/ui/render.py:27]: use double quotas in class annotation: """

File: src/hdusd/ui/render.py [hunk 2]
```diff
@@ -166,6 +201,44 @@ class HDUSD_RENDER_PT_render_settings_viewport(RenderSettingsPanel):
     engine_type = 'VIEWPORT'
 
 
+class HDUSD_RENDER_PT_help_about(HdUSD_Panel):
+    ''' Help/About UI panel '''
```
Comment (bnagirniak) [inline src/hdusd/ui/render.py:205]: same here without first and last spaces

**Review by bnagirniak — APPROVED** [review]
Everything works good, just small code improvements

File: src/hdusd/ui/render.py [hunk 3]
```diff
@@ -13,11 +13,45 @@
...
+    bl_description = "Open web page in browser"
+
+    page: bpy.props.StringProperty(name="Page")
+
+    def execute(self, context):
+        url = {
+            'main_site':     "https://www.amd.com/en/technologies/radeon-prorender",
+            'documentation': "https://radeon-pro.github.io/RadeonProRenderDocs/en/usd_hydra/about.html",
+            'downloads':     "https://www.amd.com/en/technologies/radeon-prorender-downloads",
+            'community':     "https://community.amd.com/t5/blender-discussions/bd-p/blender-discussions",
```
Comment (bsavery) [inline src/hdusd/ui/render.py:47]: I think we should point users to https://github.com/GPUOpen-LibrariesAndSDKs/BlenderUSDHydraAddon/discussions

**Pull request merged**